### DISCUSSION TITLE
Setup hard for non technical folk

### DISCUSSION THREAD

**Discussion by mlevison** [Vinzent03/obsidian-git#627] [General]
I tried explaining the beauty of this plugin to several non-technical people recently. The conversation died when they learned they would need to install Git. Have you considered using a library to the Git grunt work, even on a real computer?

**Comment by Vinzent03** [comment]
I don't plan to do this. Installing Git is the easiest part of using Git. Git is just technically, and it's hard to overcome this. That's just part of Git and it's beauty.
